.ci/configure_crio_for_kata.sh

```diff
@@ -16,14 +16,19 @@ minor_crio_version=$(crio --version | head -1 | cut -d '.' -f2)
 if [ "$minor_crio_version" -ge "12" ]; then
 	echo "Configure runtimes map for RuntimeClass feature"
 	echo "- Set runc as default runtime"
-	sudo sed -i 's!runtime_path =.*!runtime_path = "/usr/local/bin/crio-runc"!' "$crio_config_file"
-	sudo sed -i 's!runtime_type =.*!runtime_type = "oci"!' "$crio_config_file"
+	if [ -n $(grep -q "crio.runc" "$crio_config_file") ]; then
+		sudo sed -i 's!runtime_path =.*!runtime_path = "/usr/local/bin/crio-runc"!' "$crio_config_file"
+		sudo sed -i '/crio-runc/a runtime_root = "/run/runc"' "$crio_config_file"
+		sudo sed -i '/runtime_type/d' "$crio_config_file"
+	fi
 	echo "- Add kata-runtime to the runtimes map"
-	sudo sed -i '/crio-runc/a[crio.runtime.runtimes.kata]' "$crio_config_file"
-	sudo sed -i '/kata/aruntime_path = "/usr/local/bin/kata-runtime"' "$crio_config_file"
-	sudo sed -i 's!runtime_root =.*!runtime_root = "/run/vc"!' "$crio_config_file"
-	sudo sed -i '/crio-runc/a runtime_root = "/run/runc"' "$crio_config_file"
-	sudo sed -i '/crio-runc/a runtime_type = "oci"' "$crio_config_file"
+	if [ -n $(grep -q "crio.runtime.runtimes.kata" "$crio_config_file") ]; then
+		sudo sed -i '/\/run\/runc/a [crio.runtime.runtimes.kata]' "$crio_config_file"
```

Comment:
I'd never noticed this issue with `sed` before - since this isn't a search-and-replace, you can't choose your quotes, hence all the backslashes ;(